### PR TITLE
Release notes for 0.20 release

### PR DESCRIPTION
**Pull request by hoodmane**

### CODE REVIEW

File: content/posts/0.20-release.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: "Pyodide 0.12.0 release"
```
Comment (rth) [inline content/posts/0.20-release.md:2]:
```suggestion
title: "Pyodide 0.20.0 release"
```

File: content/posts/0.20-release.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+date: 2022-01-10
```
Comment (rth) [inline content/posts/0.20-release.md:3]: Need updating once we know the release date.

File: content/posts/0.20-release.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+We upgraded from Emscripten 2.0.16 to Emscripten 2.0.27. We are still a fair
+distance behind the newest Emscripten release due to a bug in Firefox versions
+95 -- 98 but fixed in Firefox 99beta. 
+
+Emscripten 2.0.27 is a major improvement over Emscripten 2.0.16, particularly
+with regards to linking: many linker errors have been changed from call-time
+errors to load-time errors or compile-time errors. In particular, this allowed
+us to fix tons of Scipy bugs.
+
```
Comment (rth) [inline content/posts/0.20-release.md:66]: It would be good to add a link to the detailed changelog somewhere

File: content/posts/0.20-release.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+## Emscripten Update
+
+We upgraded from Emscripten 2.0.16 to Emscripten 2.0.27. We are still a fair
+distance behind the newest Emscripten release due to a bug in Firefox versions
+95 -- 98 but fixed in Firefox 99beta. 
+
+Emscripten 2.0.27 is a major improvement over Emscripten 2.0.16, particularly
+with regards to linking: many linker errors have been changed from call-time
+errors to load-time errors or compile-time errors. In particular, this allowed
+us to fix tons of Scipy bugs.
```
Comment (rth) [inline content/posts/0.20-release.md:65]: Maybe say this in a more politically correct way. They are bugs when run on WASM, but not bugs regular scipy users would encounter.

File: content/posts/0.20-release.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
+---
+title: "Pyodide 0.12.0 release"
+date: 2022-01-10
+draft: true
```
Comment (rth) [inline content/posts/0.20-release.md:4]: I haven't really found a use for it. Probably better to always keep it to false?

**Review by rth — APPROVED** [review]
Thanks @hoodmane , it's a nice read!

The only thing is even if we release today/tomorrow, I think it would be better to make the announcement on Twitter on a weekday, for instance on Monday.

File: content/posts/0.20-release.md (hunk 4 above)
Comment (hoodmane) [reply content/posts/0.20-release.md:65]: Yeah, as stated the plain text implication is incorrect in addition to being rude.

File: content/posts/0.20-release.md (hunk 5 above)
Comment (hoodmane) [reply content/posts/0.20-release.md:4]: Or leave it out entirely? The `./hugo create` script adds this.

File: content/posts/0.20-release.md (hunk 3 above)
Comment (hoodmane) [reply content/posts/0.20-release.md:66]: And from the changelog to here.

File: content/posts/0.20-release.md (hunk 2 above)
Comment (hoodmane) [reply content/posts/0.20-release.md:3]: Aiming for today =)

**Comment by hoodmane** [comment]
> it would be better to make the announcement on Twitter on a weekday

So merge this whenever and make announcement on a week day? Sounds good to me. I don't make the announcements anyways.

**Comment by rth** [comment]
OK, will do on Monday.

**Pull request merged**